src/renderer/hooks/useContextMenu.ts

```diff
@@ -18,6 +19,8 @@ export const useContextMenu = (render: () => JSX.Element): UseContextMenu => {
     // eslint-disable-next-line react/hook-use-state
     const [id] = useState(createUniqueId);
 
+    const [position, setPosition] = useState({ x: 0, y: 0 });
```

Comment:
Are you even using this value? I know it's returned by the hook, but I don't see anyone using the returned value.

Reply:
I explained this on discord. Unless I just missed something, the menu doesn't get rerendered when the position changes, causing the ref to not get updated and therefore the submenu to not be positioned properly. This was a way to get this hook to rerender when the position changes. 

If you can find a better solution, please make a PR. But this was the best I could do

Reply:
turns out, since we changed this file in the breakpoints PR to save the event as state, this is no longer necessary